<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scene</class>
 <widget class="QDialog" name="Scene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Img/icon.png</normaloff>:/Img/icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>2.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lbl_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-180</y>
     <width>1161</width>
     <height>1001</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Img/back-game.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cerrar">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>580</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(255, 56, 38);
	color: white;
	border: 2px solid rgb(255, 56, 38);
	font: 63 14pt &quot;Segoe UI Semibold&quot;;
}

QPushButton:hover { 
	background-color: rgba(255, 255, 255, .1);
	color: white;
	border: 2px solid white;
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>20</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ABF0DA;
border: 0px;
font: 15pt &quot;OCR A Extended&quot;;</string>
   </property>
   <property name="title">
    <string>Player 1</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="lbl_namePlayer1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 17pt &quot;Harlow Solid Italic&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 13px;</string>
    </property>
    <property name="text">
     <string>User name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_score1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 40pt &quot;Harlow Solid Italic&quot;;
color: #ABF0DA</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 13px;</string>
    </property>
    <property name="text">
     <string>score</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>190</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ABF0DA;
border: 0px;
font: 15pt &quot;OCR A Extended&quot;;</string>
   </property>
   <property name="title">
    <string>Player 2</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 13px;</string>
    </property>
    <property name="text">
     <string>User name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_namePlayer2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 17pt &quot;Harlow Solid Italic&quot;;
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_score2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>79</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 40pt &quot;Harlow Solid Italic&quot;;
color: #ABF0DA</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 13px;</string>
    </property>
    <property name="text">
     <string>score</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>440</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #97493A;
	color: white;
	border: 2px solid #97493A;
	font: 63 14pt &quot;Segoe UI Semibold&quot;;
}

QPushButton:hover { 
	background-color: rgba(255, 255, 255, .1);
	color: white;
	border: 2px solid white;
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>490</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #97493A;
	color: white;
	border: 2px solid #97493A;
	font: 63 14pt &quot;Segoe UI Semibold&quot;;
}

QPushButton:hover { 
	background-color: rgba(255, 255, 255, .1);
	color: white;
	border: 2px solid white;
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0.1);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>350</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #3A9758;
	color: white;
	border: 2px solid #3A9758;
	font: 63 14pt &quot;Segoe UI Semibold&quot;;
}

QPushButton:hover { 
	background-color: rgba(255, 255, 255, .1);
	color: white;
	border: 2px solid white;
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_level">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 15pt &quot;Harlow Solid Italic&quot;;
color: #FFD502</string>
   </property>
   <property name="text">
    <string>Level 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_turno">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 15pt &quot;Harlow Solid Italic&quot;;
color: #FFD502</string>
   </property>
   <property name="text">
    <string>Player 1 is playing...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 18pt &quot;Harlow Solid Italic&quot;;
color: #9D1ED8;
background-color: white;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>00:00:60</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_cerrar</sender>
   <signal>clicked()</signal>
   <receiver>Scene</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>721</x>
     <y>754</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>648</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
